torch_xla_py/xla_dist.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+"""Tool to distribute training on Cloud TPU Pods."""
+from __future__ import division
+from __future__ import print_function
+
+
+class Worker(object):
+
+  def __init__(self, internal_ip, mach_type, zone, hostname=None):
+    self._internal_ip = internal_ip
+    self._mach_type = mach_type
```

Comment:
this is the only place where we distinguish tpu from vm, right?

Reply:
Yeah pretty much only the machine type. The validation logic will be the same b/w tpu and vm, since both need to be in same zone, and each cluster have same machine types.